Micromouse/Maze.cpp

```diff
@@ -56,6 +56,14 @@ bool Maze::isWall(CellCoordinate pos, Direction dir) const {
     return isWall(pos.toNode() + dir);
 }
 
+bool Maze::isWall(NodeCoordinate pos, Direction dir) const {
```

Comment:
Not necessary but I think anytime you repeatedly do an operation in the function call you might as well make it part of the function. With something as minor as `pos + dir` it is really just a matter of style at that point though.